app/controllers/search_controller.rb

```diff
@@ -122,21 +122,27 @@ def usernames
   # TODO: [@rhymes] the homepage feed uses `feed_content_search` as an index,
   # we should eventually move it to a JSON result
   # in ArticlesController#Homepage or HomepageController#show
-  # rubocop:disable Metric/PerceivedComplexity
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
   def feed_content
     class_name = feed_params[:class_name].to_s.inquiry
 
     enable_search_2_homepage = (
       class_name.Article? &&
       feed_params[:search_fields].blank? &&
+      feed_params[:sort_by].present? &&
```

Comment:
this was added to distinguish the case of "listing articles in the search page" and "listing articles in any other page". They have similar behaviours but they differ in how sort is performed, and this this the only param I found short of checking the referrer :D 

Can't wait to split all of this stuff in different endpoints 😭